src/analysis/sensitivity.jl

```diff
@@ -451,78 +455,107 @@ ADRIA.sensitivity.rsa(X, y; S=10)
    Accessible at: http://www.andreasaltelli.eu/file/repository/Primer_Corrected_2022.pdf
 """
 function rsa(
-    X::DataFrame, y::AbstractVector{<:Real}, model_spec::DataFrame; S::Int64=10
-)::NamedDimsArray
+    X::DataFrame, y::AbstractVector{<:Real}, factors::Vector{Symbol}, model_spec::DataFrame;
+    S::Int64=10,
+)::YAXArrays.Dataset
     N, D = size(X)
 
-    X_di = zeros(N)
+    X_i = zeros(N)
```

Comment:
There was a reason for X_di ($X_{d, i}$) but that may now be lost to the sands of time...

Reply:
Ah makes sense. Should I revert or leave as is?

Reply:
Leave as is